src/Microsoft.Azure.SignalR.Common/ClientInvocation/IRoutedClientResultsManager.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IRoutedClientResultsManager
```

Comment:
Considering the class name indicate it's a router manager, we can simplify the method names to remove the `Routed` word?

e.g. `AddRoutedInvocation()` -> `AddInvocation()`

Reply:
fixed